### PR TITLE
Download section & new installation FAQs

### PR DESCRIPTION
**Pull request by TobiGr**
Added a new download section, did some small refactoring of bad code (more info in the commits) and added the NewPipe legacy icon to the press kit.

image

image

I have also added (but commented out for now) a "get started" section with some links to some FAQs and tutorials. The plan is to enable it once we have more tutorials. I'd like to add some icons / graphics to it. This is how it looks:
image

I also thought about removing all borders from both ("get it" and "get started") grids and add some margin to all boxes. I could not decide, which version looks better.

I'd like to get some feedback on one the way, the link to the release post and the latest changelog are stored / processed. See the `_includes/release_data.html`. We have three options to store the data:
1. Store in `_data/release.yml`: We cannot use liquid tags to generate the URL to the blog post and would need to escape every `>` in the HTML of the changelog
2. Store data in `_includes/release_data.html` as done now. 
3. Store data in web-api: this allows us to display some info in the app, too.

When there is a blog post for the current release, there is also a "blog post" button in the "Release Highlights" section.

### To Do

- [x] Add tutorial for adding our F-Droid repo
- [ ] Add tutorial for installing NewPipe
- [x] Replace link placeholders to the tutorials mentioned above
- [x] Code clean up

@Poolitzer @gkeegan  Can you take care of the tutorials, please?

### Should be merged with #159

### CODE REVIEW

**Comment by Poolitzer** [comment]
Yup I will.

File: press/index.html [hunk 1]
```diff
@@ -16,11 +16,11 @@ <h4>About NewPipe</h4>
             <p style="margin-bottom: 15px;">A good user experience is important to the developers of NewPipe. Therefore the app offers many solutions for all possible situations. In this way, the resolution and codecs of the video and audio can be set to save data volume in mobile networks. Likewise, NewPipe's video popup and background players make it possible to seamlessly integrate videos and music into daily mobile phone use.</p>
             <p><strong>Quick information:</strong></p>
             <ul>
-                <li>Current stable version: <span class="api-github-stable-version">0.11.5 (Effective 11th February 2018)</span></li>
+                <li>Current stable version: <span data-newpipe-api="flavors.fdroid.stable.version">0.11.5 (Effective 11th February 2018)</span></li>
```
Comment (B0pol) [inline press/index.html:19]: Please update to 0.19.5…
NewPipe api uses javascript, therefore if you block it you will see this info. We should not update it every update, but it has been more than two years now.

Plus you did that for contributors and languages below.

File: press/index.html [hunk 2]
```diff
@@ -16,11 +16,11 @@ <h4>About NewPipe</h4>
...
                 <li>Initial release: 4. September 2015</li>
                 <li>Founder: Christian Schabesberger</li>
-                <li class="linebreak">Available translations: <span class="api-translations">43</span><br>You can find a list with all current supported languages at <a href="https://hosted.weblate.org/projects/newpipe/#languages" target="_blank">Weblate</a>.</li>
-                <li>Contributors: <span class="api-contributors">178</span></li>
+                <li class="linebreak">Available translations: <span data-newpipe-api="stats.translations">60</span><br>You can find a list with all current supported languages at <a href="https://hosted.weblate.org/projects/newpipe/#languages" target="_blank">Weblate</a>.</li>
+                <li>Contributors: <span data-newpipe-api="stats.contributors">440</span></li>
```
Comment (B0pol) [inline press/index.html:23]: below: last modified: June 2020

File: press/index.html (hunk 1 above)
Comment (TobiGr) [reply press/index.html]: Done

**Comment by TobiGr** [comment]
@B0pol I didn't like the shadow, too. I needed to add it because of the position of the arrows. What about that:
image

File: README.md [hunk 3]
```diff
@@ -149,3 +149,20 @@ Writing short FAQs is quite easy and there are only few things to consider.
 2. To separate parts of your article with a thin line, use `<hr/>` tags. 
 3. To insert images, use the `<img/>` tag inside of a `<figure>` tag. Additionally, each image needs a caption (`<figcaption>`).
 4. Please keep your sentences as shorts as possible. This makes it easier to follow you and your instructions.
+
+#### Usage of WEB-API
```
Comment (TheAssassin) [inline README.md:153]:
```suggestion

#### Usage of web API

```

File: README.md [hunk 4]
```diff
@@ -149,3 +149,20 @@ Writing short FAQs is quite easy and there are only few things to consider.
...
+We have an API which provides data for running the website to circumvent contacting other servers when visiting our website.
+You can find its source code [at GitHub](https://github.com/TeamNewPipe/web-api) and the served data [here](https://newpipe.schabi.org/api/data.json).
```
Comment (TheAssassin) [inline README.md:155]:
```suggestion
You can find its source code [on GitHub](https://github.com/TeamNewPipe/web-api) and the served data [here](https://newpipe.schabi.org/api/data.json).
```

File: README.md (hunk 3 above)
Comment (TheAssassin) [reply README.md:153]: Please add a note that this API is for NewPipe's own purposes only and should not be used by other people. They can set up their own instance if they want to.

Comment (TobiGr) [reply README.md:153]: > Please add a note that this API is for NewPipe's own purposes only and should not be used by other people. They can set up their own instance if they want to.

I think this note belongs in the API repo.

Comment (TheAssassin) [reply README.md]: You could say the same about the entire section.

The question is, will people actually read the API *repo* documentation if they just want to use the API themselves, now that they discovered it? I think a short note wouldn't hurt. We can add one to the API repo, too, of course.

Comment (TobiGr) [reply README.md]: I added:
> If you want to use the API for other purposes than running or developing our website, please host it yourself.

**Comment by TobiGr** [comment]
It is now possible to reference an FAQ entry in the URL like: `https://newpipe.schabi.org/FAQ/#download-signing-keys`
There is now a tutorial for adding our F-Droid repo. Thank you @gkeegan!

**Comment by B0pol** [comment]
> It is now possible to reference an FAQ entry in the URL like: https://newpipe.schabi.org/FAQ/#download-signing-keys

Wow I was looking for this exactly yesterday… IMO we should add a link icon at the right of the question

E.g.  (credits to Freepik, licensed under Flaticon license) (just an example, we should make it red)

**Comment by TobiGr** [comment]
> IMO we should add a link icon at the right of the question

good idea. I was also thinking about changing the window location / URL when a FAQ entry is clicked so one can link easier. Adding an icon is even better

**Comment by TobiGr** [comment]
I updated the release data to match version 0.19.8 and improved the documentation.

**Pull request merged**